src/main/resources/schemas/exchange/exchange.xsd

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            xmlns:org="urn:gov:hhs:fha:nhinc:exchange:directory"
+            xmlns:tns="urn:gov:hhs:fha:nhinc:exchange"
+            targetNamespace="urn:gov:hhs:fha:nhinc:exchange"
+            elementFormDefault="qualified">
+  <xsd:import namespace="urn:gov:hhs:fha:nhinc:exchange:directory" schemaLocation="organizationDirectory.xsd"/>
+
+  <xsd:element name="ExchangeInfo" type="tns:exchangeInfoType"/>
+  <xsd:complexType name="exchangeInfoType">
+    <xsd:sequence>
+      <xsd:element name="refreshInterval" type="xsd:long"/>
+      <xsd:element name="maxNumberOfBackups" type="xsd:integer"/>
+      <xsd:element name="lastUpdated" type="xsd:dateTime"/>
+      <xsd:element name="defaultExchange" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="exchanges" type="tns:exchangeListType"/>
+    </xsd:sequence>
+  </xsd:complexType>
+
+  <xsd:complexType name="exchangeListType">
+    <xsd:sequence>
+      <xsd:element name="exchange" type="tns:exchangeType" maxOccurs="unbounded"/>
+    </xsd:sequence>
+  </xsd:complexType>
+
+  <xsd:complexType name="exchangeType">
+    <xsd:sequence>
+      <xsd:element name="tLSVersions" type="tns:tLSVersionType"/>
```

Comment:
tlsVersions should be min=0.  I don't think we can get this info from the uddi right now.